src/engine/ratecontrol.h

```diff
@@ -34,6 +34,31 @@ class RateControl : public EngineControl {
     RateControl(QString group, UserSettingsPointer pConfig);
     virtual ~RateControl();
 
+    // Enumerations which hold the state of the pitchbend buttons.
+    // These enumerations can be used like a bitmask.
+    enum RATERAMP_DIRECTION {
```

Comment:
normally we use camel case for enum names

Reply:
Yes, I just copy and pasted these without cleaning up the legacy code style yet.

Reply:
The amount of technical and design debt in the preferences is pretty overwhelming. The whole system could use a cleanup.